MapMaker/wwwroot/js/tools.js

```diff
@@ -5,6 +5,20 @@ var layerSizes = [[99, 0], [199, 100], [299, 200], [399, 300], [499, 400], [599,
 
 paper.install(window);
 
+var downloadAsSVG = function (fileName) {
```

Comment:
This should probably go in its own js file, rather than the tools file. That way, we can update the two independently without creating issues.